update communities spec as per content topic changes (#118)
vacp2p/rfc-index: status/56/communities.md

contentTopic = "/waku/1/0x" + topic + "/rfc26"
```

#### Community Shard Info

If a community is assigned a dedicated shard then the shard info for that community is published on a content topic derived from a specialized key. This is useful for users joining the new community so that they can subscribe to this specific content topic.

```js
chatID = publicKey+"-shard-info"
hash = hex(keccak256(encodeToHex(chatID))[2:])

topicLen = 4
if len(hash) < topicLen {
    topicLen = len(hash)
}
var topic [4]byte
for i = 0; i < topicLen; i++ {
    topic[i] = hash[i]
}

contentTopic = "/waku/1/0x" + topic + "/rfc26"
```

#### Community channels/chats

> All channels/chats shall use a single content-topic which is derived based on a universal chat id ir-respective of their individual unique chat ids.

Reviewer: ```suggestion
All channels/chats shall use a single content-topic which is derived from a universal chat id irrespective of their individual unique chat ids.
```
